### PR TITLE
[LibreElec] design fit to AZR - part2

### PR DESCRIPTION
**Pull request by Nanomani**

### CODE REVIEW

**Comment by Nanomani** [comment]
I m not sure for media storage if it is good ?
I have put all PNG in media/elec/*.png

**Comment by beatmasterRS** [comment]
Looks stunning! Very good work! Can't wait to try the corelec settings window :D

**Comment by Nanomani** [comment]
absolutely consistent : 

screenshot00000
screenshot00001
screenshot00002
screenshot00003
screenshot00004
screenshot00005

**Comment by beatmasterRS** [comment]
Looks good and i can see, that the textbox is gone and the label has moved to the top line, very nice! :)

**Comment by beatmasterRS** [comment]
Please commit your changes :)

**Comment by Nanomani** [comment]
screenshot00009
screenshot00010

**Comment by Nanomani** [comment]
screenshot00009
screenshot00011

**Comment by Nanomani** [comment]
Some little adjustment are necessary with focus (a bite too large)
But sometime find the good place to do the good code make me crazy :D

**Comment by beatmasterRS** [comment]
The background on the 2nd screen has less width then the 1st screen

File: 1080i/service-LibreELEC-Settings-mainWindow.xml [hunk 1]
```diff
@@ -9,15 +9,16 @@
...
             <include>Animation.Common</include>
             <left>72</left>
             <right>72</right>
-            <top>215</top>
+            <top>208</top>
             <height>656</height>
             <control type="image">
                 <left>-8</left>
-                <right>0</right>
+                <right>8</right>
```
Comment (beatmasterRS) [inline 1080i/service-LibreELEC-Settings-mainWindow.xml:21]: Thats the reason for the focus issue, i think..

**Comment by Nanomani** [comment]
can be merge :)
thanks

**Pull request merged**